jobbergate-cli/jobbergate_cli/exceptions.py

```diff
@@ -132,3 +132,17 @@ def wrapper(*args, **kwargs):
             ) from err
 
     return wrapper
+
+
+def handle_errors(func):
```

Comment:
Since this is only being used in one place, I think it makes sense to just move the logic into `safe_entrypoint()`. Adding an extra layer here doesn't add much I think.

Reply:
`handle_errors` is actually on the tests to ensure we got it covered when testing the commands: https://github.com/omnivector-solutions/jobbergate/blob/00a99988bbe05a3b4f88e157892f499b9d231923/jobbergate-cli/tests/subapps/conftest.py#L28-L40

That is why it is not directly on `safe_entrypoint`.

Reply:
it seems like you could also just include the two existing decorators here to avoid having to declare an extra decorator:

```
 @pytest.fixture 
 def make_test_app(dummy_context): 
     def _main_callback(ctx: Context): 
         ctx.obj = dummy_context 
  
     def _helper(command_name: str, command_function: Callable): 
         main_app = Typer() 
         main_app.callback()(_main_callback) 
         safe_command = handle_abort(handle_authentication_error(command_function))
         main_app.command(name=command_name)(safe_command) 
         return main_app 
```

Reply:
Suggestion addressed in [e50914c](https://github.com/omnivector-solutions/jobbergate/pull/587/commits/e50914ce8b07429de28e6aa70311abb22f9877fc).